pyproject.toml

```diff
@@ -22,14 +23,15 @@ jaxtyping = "^0.2.8"
 typer = "^0.7.0"
 lazy-loader = "^0.2"
 matscipy = "^0.8.0"
-znh5md = "^0.1.7"
+znh5md = "^0.2"
 pydantic = "^2.3.0"
 jax = "^0.4.25"
-zntrack = {version = "^0.7.2", optional = true}
-mlflow = {version = "^2.11.3", extras = ["mlflow"]}
+zntrack = {version = "^0.7.3", optional = true}
+mlflow = {version = "^2.0.0", optional = true}
```

Comment:
Can you make it `^2` and not `^2.0.0` which will restrict it to `mlflow 2.0.X`

Reply:
sure